web/src/tippyjs.ts

```diff
@@ -643,6 +644,24 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ["#recent_view .recipient_bar_icon", "#inbox-view .recipient_bar_icon"].join(","),
+        delay: LONG_HOVER_DELAY,
+        appendTo: () => document.body,
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const current_visibility_policy_str = $elem.attr("data-tippy-content");
+            instance.setContent(
+                ui_util.parse_html(
+                    render_change_visibility_policy_button_tooltip({current_visibility_policy_str}),
+                ),
+            );
+        },
+        onHidden(instance) {
+            instance.destroy();
+        },
```

Comment:
Let's export these as `topic_visibility_tooltip_prop` and export them to be used in `message_list_tooltip`.